packages/riverpod_cli/fixtures/notifiers/golden/pubspec.yaml

```diff
@@ -9,6 +9,6 @@ environment:
 dependencies:
   flutter:
     sdk: flutter  
-  flutter_riverpod: ^1.0.0-dev.0
-  hooks_riverpod: ^1.0.0-dev.0
-  riverpod: ^1.0.0-dev.0
\ No newline at end of file
+  flutter_riverpod: ^0.14.0
+  hooks_riverpod: ^0.14.0
```

Comment:
The CI overrides these version to use the latest, so this won't work.

The reason being, to test the tool before publishing

Reply:
Will merge anyway, thanks!